### DISCUSSION TITLE
Showing compute and last saved information in the toolbar.

### DISCUSSION THREAD

**Discussion by SiddhanthUnnithan** [microsoft/vscode-jupyter#9605] [New Ideas]
Would it be possible to show CPU and RAM usage as well as the "last saved" time checkpoint? Azure ML users are able to view this information through the Az Nb component underneath their core toolbar:

![image](https://user-images.githubusercontent.com/6413608/162023213-861ebfc3-75ab-442f-83b5-08e2ef076ce4.png)

For the Azure ML remote-connect scenario we're already persisting notebook context and eventually would like to persist file context. If we want to be consistent with the information shown to users across the two products, I believe we should also show compute and last-saved information. The assumption I'm making here is that if Az Nb has a way to surface the information, we should be able to do the same in VS Code.

**Comment by DonJayamanne** [comment]
Not sure about `last saved a few seconds`, feels like that conflicts (duplicates) with vs codes `save/auto save` functionality. I.e. if auto save is enabled then not sure we need to display the fact that the file was auto saved.

As for CPU & RAM, I'm not sure this would be possible for remote Jupyter servers, would be possible as along as its an VS Code remote connection or running locally. But feel this could be a candidate for the Jupyter power toys. We haven't any user ask for this so far.
